spec/integration/admin/reports/reports_spec.rb

```diff
@@ -133,6 +134,13 @@
       expect(page).to have_content(submission2.program.name)
       expect(page).to have_content(submission1.admin_notes)
       expect(page).not_to have_content(submission3.program.name) if current_partner.graduate?
+      expect(page).not_to have_content("Professor Thesis Advisor Test") if current_partner.graduate?
```

Comment:
On line 126 above, we need to tag this test like this:
`it 'displays the Custom Report page and allows filtering by semester', graduate: true, honors: true do`